spec/models/event_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe Event, type: :model do
+  let(:event) {
+    Event.new(title: 'Minsk.rb best',
+              place_title: 'Eventspace.by',
+              place_cord: 'Oktyabrskaya str. 16a, Minsk',
+              description: 'Fixie tote bag ethnic keytar. Neutra vinyl American...',
+              start_at: Time.now,
+              end_at: Time.now + 1.week)
+  }
+
+  it 'is valid with valid attributes' do
+    expect(event).to be_valid
+  end
+
+  it 'is not valid without a title' do
+    event.title = nil
+    expect(event).to_not be_valid
+  end
+
+  it 'is not valid when the length of :title is less than 6' do
+    event.title = 'Minsk'
+    expect(event).to_not be_valid
+  end
+
+  it 'is not valid when the length is more than 20' do
+    event.title = 'We are Minsk.rb(EEE BOY)'
```

Comment:
In general, it's totally fine.
I just _putting forward_ that it may be easier to initialize these strings as string multiplying:
`event.title = 'a' * 21`

But on other hand, it's a _magical_ numbers, so maybe @alexshgov have an other opinion on that.